agent/api/task/task.go

```diff
@@ -76,6 +76,16 @@ const (
 	// awslogsCredsEndpointOpt is the awslogs option that is used to pass in an
 	// http endpoint for authentication
 	awslogsCredsEndpointOpt = "awslogs-credentials-endpoint"
+
+	// ipcModeNone specifies the string used to define the `none` docker IPC resource namespace sharing mode
```

Comment:
Any reason that `ipcModeNone` is not grouped with other pid/ipc constants?

Reply:
Just to distinguish that None is an option for IPC but thinking again, it makes more sense to just group it and leave out the comment.